site/site.go

```diff
@@ -102,7 +102,8 @@ func New(opts *Options) *Handler {
 		// Set ETag header to the SHA1 hash of the file contents.
 		name := filePath(r.URL.Path)
 		if name == "" || name == "/" {
-			// Serve the directory listing.
+			// Serve the directory listing. This intentionally allows directory listings to
+			// be served. This file system should not contain anything sensitive.
 			http.FileServer(opts.BinFS).ServeHTTP(rw, r)
```

Comment:
It feels beneficial to keep this for listing what binaries are supported. Thoughts?

![Screenshot from 2024-02-20 09-37-57](https://github.com/coder/coder/assets/5446298/b6743eca-08fa-46c7-aeb3-3609bbe5ba80)